_includes/navbar.html

```diff
@@ -42,14 +42,16 @@
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community<strong class="caret"></strong></a>
                     <ul class="dropdown-menu">
                         <li><a href="/community/">Get In Touch!</a></li>
-                        <li><a href="http://www.meetup.com/jclouds/">Attend An Event</a></li>
                         <li><a href="/community/users/">Who Is Using jclouds?</a></li>
                         <li class="divider"></li>
                         <li class="dropdown-header">Contribute</li>
                         <li><a href="/reference/report-a-bug/">Report a Bug</a></li>
-                        <li><a href="http://jclouds.ci.cloudbees.com">Continuous Integration</a></li>
+                        <li><a href="https://builds.apache.org/view/J/view/jclouds/">Continuous Integration</a></li>
                         <li><a href="https://cwiki.apache.org/confluence/display/JCLOUDS/How+to+Contribute">How To Contribute Code</a></li>
                         <li><a href="https://cwiki.apache.org/confluence/display/JCLOUDS/How+to+Contribute+Documentation">How To Contribute Documentation</a></li>
+                        <li class="divider"></li>
+                        <li class="dropdown-header">Apache Events</li>
+                        <li><a href="https://www.apache.org/events/current-event.html"><img src="https://www.apache.org/events/current-event-234x60.png"/></a></li>
```

Comment:
Would this perhaps be better placed in the `About Apache` menu? It doesn't really seem obviously related to the **jclouds** community as such?